tachyon/crypto/hashes/prime_field_serializable.h

```diff
@@ -69,6 +71,28 @@ class PrimeFieldSerializable<std::vector<T>> {
   }
 };
 
+template <typename T, size_t N>
+class PrimeFieldSerializable<absl::InlinedVector<T, N>> {
```

Comment:
We need to discuss whether the name of the object is appropriate.
ex) PrimeFieldConvertible?

Reply:
As mentioned in #116, since it's out of scope of this PR, I will fix in a near future.